cypress/pageobject/Patient/PatientDetails.ts

```diff
@@ -0,0 +1,46 @@
+export class PatientDetailsPage {
+  clickAssignToVolunteer() {
+    cy.contains("button", "Assign to a Volunteer")
```

Comment:
```suggestion
    cy.contains("button", "Assign to a Volunteer")
```

use proper Id for the button

Reply:
check for cy.verifyAndClickElement() function after you give them the id